oat-sa/extension-lti-outcomeui
# Fix/ms 469/missing highlight removal ACs

**Ticket:** https://oat-sa.atlassian.net/browse/MS-469

## What has changed?

- Added toggle ring
- Added hover effect
- Added eraser icon

## How to test?

- Login as a scorer
- Try using an eraser without highlights, it should not turn on
- Make some highlights
- Turn the eraser on, you'll see a toggle ring around the eraser
